Gaijinizer patch/Map209.txt

```diff
@@ -93,8 +99,10 @@ impossible......#MTLed
 テープで言っていたように……
 僕を殺してくれ…………！！！
 > CONTEXT: Map209/events/1/pages/0/66/Dialogue/kazuya
-I have a request ......... Reina .........
-As you said with tape...Kill me ............! !
+I have a request, Reina...
+
+Like you said you'd do on the tape...
+Please kill me...!!
 !#MTLed
 > END STRING
```

Comment:
Need to clean up the end here and add your credit. Additionally "the tape" -> "that tape".